node/src/main/java/eu/cloudnetservice/node/service/defaults/AbstractService.java

```diff
@@ -498,9 +498,17 @@ protected void executeDeployment(@NonNull ServiceDeployment deployment) {
         var fileName = Files.isDirectory(path)
           ? path.getFileName().toString() + '/'
           : path.getFileName().toString();
-        // check if the file is ignored
-        return deployment.excludes().stream().noneMatch(pattern -> fileName.matches(pattern.replace("*", "(.*)")))
-          && !DEFAULT_DEPLOYMENT_EXCLUSIONS.contains(fileName);
+        // file matcher escaping asterix chars for regex
+        Predicate<String> fileMatcher = input -> fileName.matches(input.replace("*", "(.*)"));
```

Comment:
I think we can move that predicate into a static final variable. Maybe we should move the `*` -> `(.*)` replacement out of this method into the command and apply the pattern fixer to it there? (Can be done in a later step after this pr as well)